lib/error.js

```diff
@@ -84,27 +84,29 @@ export const makeError = ({
 	error.shortMessage = shortMessage;
 	error.command = command;
 	error.escapedCommand = escapedCommand;
+	error.cwd = cwd;
+
+	error.failed = true;
+	error.timedOut = Boolean(timedOut);
+	error.isCanceled = isCanceled;
+	error.isTerminated = signal !== undefined;
 	error.exitCode = exitCode;
 	error.signal = signal;
 	error.signalDescription = signalDescription;
```

Comment:
Boolean result properties like `timedOut` come before more descriptive ones like `exitCode` and `signal`.